### PR TITLE
feat: add theme setup and fonts

### PR DESCRIPTION
**Pull request by abhishek-gkmit**
- add `useStyles` hook
- add `useThemeContext` hook to use `ThemeContext`
- add fonts `Nunito Sans` and `Montserrat`
- add font constants
- add colors
- add some default global styles

### CODE REVIEW

File: src/config/contexts/ThemeContext.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { createContext, useMemo } from 'react';
+import { useColorScheme } from 'react-native';
+
+import { colors } from '@theme/colors';
+
+const ThemeContext = createContext<ThemeContextValues>({
+  theme: 'light',
```
Comment (02priyanshi) [inline src/config/contexts/ThemeContext.tsx]: use constant for this

File: src/constants/fonts.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+const fontFamily = {
+  NunitoSansExtraLight: 'NunitoSans_10pt-ExtraLight',
+  NunitoSansExtraLightItalic: 'NunitoSans_10pt-ExtraLightItalic',
+  NunitoSansLight: 'NunitoSans_10pt-Light',
+  NunitoSansLightItalic: 'NunitoSans_10pt-LightItalic',
+  NunitoSansRegular: 'NunitoSans_10pt-Regular',
+  NunitoSansRegularItalic: 'NunitoSans_10pt-RegularItalic',
+  NunitoSansMedium: 'NunitoSans_10pt-Medium',
```
Comment (02priyanshi) [inline src/constants/fonts.ts:8]: why using two font families?

File: src/theme/globalStyles.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { StyleSheet } from 'react-native';
+
+function getThemedStyles(colors: Colors) {
+  return StyleSheet.create({
+    screen: {
+      flex: 1,
+      backgroundColor: colors.background,
+    },
+
+    flexContainer: {
```
Comment (02priyanshi) [inline src/theme/globalStyles.ts]: use proper names

**Review by 02priyanshi — CHANGES_REQUESTED** [review]

File: src/constants/fonts.ts (hunk 2 above)
Comment (abhishek-gkmit) [reply src/constants/fonts.ts:8]: One for special heading text and one for text throughout the app

File: src/theme/globalStyles.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ function getThemedStyles(colors: Colors) {
       backgroundColor: colors.background,
     },
 
-    flexContainer: {
+    flexJustifiedAlignedCenter: {
```
Comment (02priyanshi) [inline src/theme/globalStyles.ts]: instead this `centerAlignContainer`

**Review by 02priyanshi — APPROVED** [review]

**Pull request merged**